src/Sylius/Component/Core/Model/Product.php

```diff
@@ -203,6 +203,10 @@ public function hasTaxon(BaseTaxonInterface $taxon)
      */
     public function getPrice()
     {
+        if (null === $this->getMasterVariant()) {
+            return 0;
```

Comment:
Yes, it is, if we still want to index product with price in _SearchBundle_... Should I change a _SearchBundle_ logic?